src/js/themes/base.js

```diff
@@ -1169,6 +1169,11 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // extend: undefined,
       // disabled: { opacity: undefined },
     },
+    tip: {
+      // container: {}, // any Box props
+      content: {}, // any Box props
+      drop: {}, // any props for the drop
+    },
```

Comment:
I'm still thinking whether there is a need for `extend`, I don't see a need at this point hence I'll wait to see if the need arises.